source/mainnet/net/mobile-wallet/add-baker-mw.rst

```diff
@@ -323,10 +317,100 @@ Add baker
+            :width: 40%
+
+    5. Review the commissions paid to bakers. When you open your baker as a pool, you earn commissions of stake delegated to your pool from other accounts. Click **Continue**.
+
+        .. image:: ../images/browser-wallet/add-baker-comms.png
+            :alt: screen showing commission amounts for bakers in read-only
+            :width: 40%
+
+    6. If you open a baker pool, you can optionally enter a URL with information about your baker to give delegators more information about your baker pool to help them research baker pools. This information is not shared in the case of a closed baker pool. Click **Continue**.
+
+        .. image:: ../images/browser-wallet/add-baker-url.png
+            :alt: screen with text box to enter url for baker pool information
+            :width: 40%
+
+    7. You have to export the baker keys so that you can start the node with the baker keys. Click **Export baker keys** and navigate to the place on your device where you want to save the file. Give the file a name and the extension .json. Click **Continue** to complete the transaction.
+
+        .. image:: ../images/browser-wallet/add-baker-export-keys.png
+            :alt: screen with buttons to show keys or export baker keys
+            :width: 40%
+
+        .. image:: ../images/browser-wallet/add-baker-continue-after-export.png
+            :alt: screen with buttons to continue after export and wallet screen dimmed behind
+            :width: 40%
+
+    .. Warning::
+
+        This is the only time that you can export the credentials. If you're going to transfer the baker keys to someone else, make sure to do so through a secure channel. Generate new keys if you believe the keys have been compromised or lost.
+
+    7. Once you have saved the keys, you see an overview screen of the add baker transaction. Review the information then click **Send**.
+
+        .. image:: ../images/browser-wallet/add-baker-review.png
+                :width: 40%
+
+    8. The wallet shows that the transaction has been submitted to the chain. Click **Finish**.
+
+        .. image:: ../images/browser-wallet/add-baker-finish.png
+                :width: 40%
+
+    9. You need to import your baker keys file to your node in order to start baking. It is preferable to update them on the node as close to the next :ref:`pay day<glossary-pay-day>` as possible to prevent the node from being down as a baker for a longer time.
+
+        - :ref:`Windows<baker-windows>`
+        - :ref:`macOS<baker-macos>`
+        - :ref:`Docker<baking-docker>`
+        - :ref:`Ubuntu<baker-ubuntu>`
+
+    Once the transaction is finalized after importing your baker keys to the node, you see the baking status reflected in the account list and on the account card.
+
+    .. image:: ../images/mobile-wallet/account-list-baking.png
+        :width: 40%
+
+    .. image:: ../images/mobile-wallet/account-details-baking.png
+        :width: 40%
+
```

Comment:
We have added this in the browser wallet in 1.0.1, so you don't have to use mobile wallet images :upside_down_face: